polytope/polytope.py

```diff
@@ -1292,25 +1301,26 @@ def volume(polyreg):
         N = 50
     elif n == 2:
         N = 500
-    elif n ==3:
+    elif n == 3:
         N = 3000
     else:
         N = 10000
 
     l_b, u_b = polyreg.bounding_box
-    x = np.tile(l_b,(1,N)) +\
-        np.random.rand(n,N) *\
-        np.tile(u_b-l_b,(1,N) )
+    x = np.tile(l_b, (1, N)) +\
+        np.random.rand(n, N) *\
+        np.tile(u_b-l_b, (1, N))
     aux = np.dot(polyreg.A, x) -\
```

Comment:
I am happy with both the single line style and the formatting proposed by @johnyf [above](https://github.com/tulip-control/polytope/pull/33#discussion_r96332166). If both of you do not find agreement, then we can just skip this change as something separate from the PR.